anderson_2007

nmehta2 opened the pull request:
changed [PARAM] values to anderson  et al paper values

dpastoor commented on models/andeson_2007.cpp:
```diff
@@ -0,0 +1,48 @@
+D_CENT = 1; 
+double NORM_WT = 70;
+double NORM_PMA = 34.8;
+double CLi = 3.83*pow(wt/NORM_WT,0.75)*(pow(pma, 3.68)/(pow(pma, 3.68) + pow(33.3, 3.68)))*(516*EXP(0.00823*((PMA-40)/52-40)))/scr)/6)*Fventilation;
```
Remember there is no EXP - instead you need to transform it to a `pow` statement

dpastoor replied:
Also, you have defined Fventilation, but you also need to make that a parameter so that can be set

dpastoor commented on models/andeson_2007.cpp:
```diff
@@ -0,0 +1,48 @@
+double NORM_WT = 70;
+double NORM_PMA = 34.8;
+double CLi = 3.83*pow(wt/NORM_WT,0.75)*(pow(pma, 3.68)/(pow(pma, 3.68) + pow(33.3, 3.68)))*(516*EXP(0.00823*((PMA-40)/52-40)))/scr)/6)*Fventilation;
+double Vi = 39.4*(wt/NORM_WT)*Finotrope;
```
same thing here Finotrope should be a parameter since we have to set it

dpastoor commented on models/andeson_2007.cpp:
```diff
@@ -0,0 +1,48 @@
+  wt  : Weight (kg)
+  pma : post-menstral age (wk)
+  scr : serum creatinine (mg/dL)
+  RDV : observed dv value (mg/L)
```
we should also be outputing Finotrope and Fventilation since we'd likely want to pass those along as well.

dpastoor reviewed (changes requested):
This is a good first start! Two things, 

1) please address the changes in structure and parameters

2) We'll want to actually find/use the 'true' random effect and residual variability terms

nmehta2 committed:
anderson_2007

changed [PARAM] values to anderson  et al paper values

nmehta2 committed:
made changes to anderson, started lo

- anderson updated
- lo is not complete (just copied and pasted anderson, haven’t changed
much code yet)

dpastoor commented:
So remember the whole point of pull requests is to keep them open until everything is cleared

Unfortunately there are still a number of errors in the model that need to be fixed before it should have been merged, so we need to go back and revert this merge
